### DISCUSSION TITLE
在dumi项目中使用less的变量语法，打包后的产物貌似无法识别

### DISCUSSION THREAD

**Discussion by Liyi-up** [umijs/dumi#2099] [General]
<img width="1512" alt="image" src="https://github.com/umijs/dumi/assets/59378475/582910f0-8e5d-4ce2-a650-65b924f5afd2">
最终解析成了.react-lib-ly-@{classPrefix}-selected ，变量这块并没有被解析成功，导致样式失效
这是我的代码仓库：https://github.com/Liyi-up/RLib

**Comment by Liyi-up** [comment]
是dumi dev 运行时编译的less产物并没有将变量语法解析

**Comment by PeachScript** [comment]
先用 Less 的 Playground 验证下你的写法：https://lesscss.org/less-preview
